src/Modal/Modal.jsx

```diff
@@ -29,7 +31,30 @@ class Modal extends Component {
   }
 
   render() {
-    const { onClose, role, ariaLabel, children } = this.props;
+    const {
+      onClose,
+      onCancel,
+      onSubmit,
+      role,
+      ariaLabel,
+      children,
+      submitText,
+      cancelText
+    } = this.props;
+
+    if (onCancel && !cancelText) {
+      // eslint-disable-line
+      console.warn(
+        "You might need to add property 'cancelText' if you need the button responsible for cancel to have text"
+      );
+    }
+    if (onSubmit && !submitText) {
+      // eslint-disable-line
+      console.warn(
+        "You might need to add property 'submitText' if you need the button responsible for submit to have text"
```

Comment:
please remove the warning - its a total valid case to not have the buttons

Reply:
the warning is not to inform you that you might not have button. this is 100 percent valid feature.
When one uses the component like 
```
             <Modal
              onClose={() => toggleModal(false)}
              onCancel={() => alert("I can cancel something")}
              onSubmit={() => alert("I can submit something")}
            >
              A very basic modal that wont scroll the background when opened.
            </Modal>
```
it is very likely that the user will forget to add 
```
  submitText="Submit"
  cancelText="Cancel"
```
properties that are for translated text. I would not like to go to storybook and I would expect a well made library to tell me what I should add to have the text.

Reply:
well made lib will have well made defaults

Reply:
the library dont translate... we need to parse translatable variables from our applications